library/template/config/default.js

```diff
@@ -3,4 +3,7 @@ module.exports = {
     Warning: do not use this to set values that differ in each environment,
     only use this for configuration that is the same across all config environments
   */
+  rollbar: {
+    accessToken: 'get an access token at rollbar.com',
```

Comment:
use `post_client_item` as key name, in some projects we also need `post_server_item`